MIGRATION.md

```diff
@@ -1,5 +1,16 @@
 # Migration
 
+## 3.1.2 → 4.0.0
+
+Drop Node.js v12 support.
+
+Please make sure you have the right version (>=14.16) of Node.js installed.
```

Comment:
Why the 14.16 exactly? Does not it work with 14.15, 14.14, etc? 

If so, should not we add `engines` to package.json?

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines

And should not we set our CI to `14.16` instead of `14.x`?

Reply:
Oh, sorry, I see `engines` in the package.json. I didn't notice it at first.

What about 14.x in the CI settings? It the tests work fine, why do we force 14.16 though?

Reply:
Roger that.